androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/Utils.java

```diff
@@ -99,4 +102,41 @@ public static Bitmap convertDrawableToBitmap(Context context, int drawableId) {
         drawable.draw(canvas);
         return bitmap;
     }
-}
\ No newline at end of file
+
+    /**
+     * Verifies if the application has the required meta-tag, `web_manifest_url`, and the
+     * `web_app_manifest.json` file, required to enable the Trusted Web Activity on Chrome OS.
+     * Important: The content the meta-tag and the JSON fire are not validated.
+     */
+    public static boolean hasChromeOsSupport(Context context) {
+        // We first check if /app/res/web_app_manifest.json is available
+        int id = context.getResources().getIdentifier(
+                "web_app_manifest.json", "raw", context.getPackageName());
+        // If an attribute with that name doesn't exist the result is 0 (which is an invalid ID for
+        // attributes)
+        if (id == 0) {
+            Log.e("ChromeOS-Support", "Could not find '/res/raw/web_app_manifest.json'");
+            return false;
+        }
+
+        // Then, we check if the 'web_manifest_url' meta-tag has been defined.
+        try {
+            ApplicationInfo applicationInfo = context.getPackageManager()
+                    .getApplicationInfo(context.getPackageName(), PackageManager.GET_META_DATA);
+            String webManifestUrl = applicationInfo.metaData.getString("web_manifest_url");
+            if (webManifestUrl == null) {
+                Log.e("ChromeOS-Support",
+                        "Could not the 'web_manifest_url' meta-tag in 'AndroidManifest.xml'.'");
+            }
+            context.getPackageManager().getResourcesForApplication(context.getPackageName());
+            return false;
```

Comment:
Do you want to return false here - this is the happy case? The logic seems a bit off.

Reply:
Yeah, changed this a bit. 

I was planning to add code to check for the actual content of the meta-tag while making the draft (then, obviously forgot about it when making this an actual PR).

I'm just checking for `bundle.containsKey()` now instead retrieving the value and returning false if the key is not there. Otherwise, return true.